docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -73,8 +73,13 @@ TEKU_DATA_PATH=/home/me/me_node
 data-path: "/home/me/me_node"
 ```
 
-The path to the Teku data directory. The default is the directory in which Teku is installed
-or `/opt/teku/database` if using the Teku Docker image.
+The path to the Teku data directory. The default directory is OS dependant:
+
+* macOS: `~/Library/teku/data`
+* Unix/Linux: `~/.teku/data`
```

Comment:
so this one is not so simple...
If XDG_DATA_HOME is set, it will be $XDG_DATA_HOME/teku, otherwise it'll be ~/.local/share/teku

Reply:
Cool, updated it to:
Unix/Linux: `$XDG_DATA_HOME/teku` if `$XDG_DATA_HOME` is set; otherwise `~/.local/share/teku`